### ISSUE TITLE
Build fails because of swift version

### ISSUE THREAD

**Issue by phiasco12** [phonegap/build#652]
Hi,

my build fails with this error:

**The “Swift Language Version” (SWIFT_VERSION) build setting must be set to a supported value for targets which use Swift. Supported values are: 3.0, 4.0, 4.2. This setting can be set in the build settings editor.**

I added this to my config.xml to no avail:

```
<preference name="UseLegacySwiftLanguageVersion" value="true" />
<preference name="swift-version" value="3.2"/>
```

and this 

```
<preference name="UseLegacySwiftLanguageVersion" value="true" />
<preference name="swift-version" value="3.0"/>
```

can someone please advice on this?

**Comment by phiasco12** [comment]
6 days and still no one?! is PGB abandoned all together then?!?  whats going on?

**Comment by phiasco12** [comment]
why no one from phonegap reply's? what's wrong with phonegap?!??!?!

**Comment by andersborgabiro** [comment]
This is primarily for bugs and suggestions for improvements.
Have you tried asking at https://forums.adobe.com/community/phonegap/build
(I'm not affiliated with Adobe in any way, but I use PGB)

**Comment by berto-nts** [comment]
Yes, I have asked in that forum, but nobody answers ... what is happening?

**Comment by andersborgabiro** [comment]
I'm as much in the dark as you :). Thankfully I haven't run into a build wall yet.

**Comment by andersborgabiro** [comment]
Hmm... Hopefully a mishap.

![phonegap](https://user-images.githubusercontent.com/1548064/59226157-acccb280-8bd2-11e9-950b-02eddcce82d5.png)

**Comment by phiasco12** [comment]
Honestly I don't know why no one from PG even bother to reply!! This is beyond unacceptable!!

Even if its just to acknowledge that they know about it and looking into it!

**Comment by danzen** [comment]
I'll see if I can prompt some response somehow...
